packages/shorebird_cli/lib/src/artifact_manager.dart

```diff
@@ -40,12 +40,19 @@ class ArtifactManager {
 
     final tempDir = await Directory.systemTemp.createTemp();
     final diffPath = p.join(tempDir.path, 'diff.patch');
-
-    await patchExecutable.run(
-      releaseArtifactPath: releaseArtifactPath,
-      patchArtifactPath: patchArtifactPath,
-      diffPath: diffPath,
-    );
+    if (File(updaterTools.path).existsSync()) {
```

Comment:
Super minor, opitinal, nit, can we extract this `File` to a variable?

Reply:
Updated the function to accept Files, so this is not longer valid.

Reply:
But if `updaterTools` is a file itself, can't we just do `updaterTools.existsSync()` instead?